<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidgets</class>
 <widget class="QWidget" name="mainWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>相框水印设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="baseConfigLayout">
     <item>
      <layout class="QVBoxLayout" name="baseSeetingLayout" stretch="0,0,0,0,0,0,0">
       <item>
        <widget class="QLabel" name="fontLabel">
         <property name="text">
          <string>字体</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFontComboBox" name="fontComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="boxSizeLabel">
         <property name="text">
          <string>边框比例</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="boxSizeComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>logo选择</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="logoComboBox"/>
       </item>
       <item>
        <widget class="QCheckBox" name="addFrameCheckBox">
         <property name="text">
          <string>添加相框</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="ioLayout">
       <item>
        <layout class="QHBoxLayout" name="InputLayout">
         <item>
          <widget class="QPushButton" name="inputButton">
           <property name="text">
            <string>输入文件夹</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lnputEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="outputLayout">
         <item>
          <widget class="QPushButton" name="outputButton">
           <property name="text">
            <string>输出文件夹</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="outputEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="configLayout">
     <item>
      <layout class="QHBoxLayout" name="leftTopConfigLayout" stretch="0,1,4">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>左上设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="leftTopComboBox">
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leftTopEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="leftBottomConfigLayout" stretch="0,1,4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>左下设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="leftBottomComboBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="leftBottomEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="rightTopConfigLayout" stretch="0,1,4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>右上设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="rightTopcomboBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="rightTopEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="rightBottomConfigLayout" stretch="0,1,4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>右下设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="rightBottomComboBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="rightBottomEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="workStatus">
       <property name="text">
        <string>处理未开始</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
